src/components/homePage/homePage.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+import SearchBar from '../searchBar/searchBar';
+import Cards from './cards/cards';
+
+const homePage = (props) => {
+  const { data } = props;
+  const [searchText, filteredText] = useState('');
+  const handleSearch = (e) => {
+    filteredText(e.target.value);
+  };
+  const countriesList = data.filter((el) => el[1].id.includes(searchText))
+    .map((el) => (
+      <Cards key={el[1].id} data={el} />
+    ));
+  return (
+    <div className="container-fluid">
+      <SearchBar value={searchText} handler={handleSearch} />
```

Comment:
- [ ] OPTIONAL: The SearchBar is totally not functional. Here you have only a handler method that updates the state when the input value is changed. There should be an onClick (in case of a submit button is present, for which the label `search` should have been a button), or an onSubmit method in case of a form. I made this comment optional because it was not mentioned by the previous reviewer. Kindly fix it in your spare time. :pray: 

![Screenshot from 2021-09-19 02-15-22](https://user-images.githubusercontent.com/64894927/133907429-60c9789f-ced2-4bcf-945f-5329a89b6a28.png)